### PR TITLE
Volvo pr Ciel day 14 part 1

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Ciel/Day 14/14_01A_CIEL14A_4.txt [hunk 1]
```diff
@@ -227,7 +227,7 @@ Unbelievable...
 [sha:3bd31c6eb2ebb4a126618fea77774a9d4e3b1e88]{
 -- Page 113, Offset 37930.
 -- 人間社会との関わり。
-She had an understanding with human society.
+She had an understanding of human society.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01A_CIEL14A_4.txt:230]: Original was correct - to have an understanding with = to have an implicit contract with

File: script/Ciel/Day 14/14_01_CIEL14_1B.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ Each and every building...
 [sha:2cacdfc9f05cc4d3282f49d750d7fc09be635951]{
 -- Page 10001, Offset 37140.
 -- 建物という建物にかかる、蜘蛛の糸のような繭。
-Each and every structure is cocooned in what looks like spider's silk.
+Each and every structure is cocooned in what looks like spider silk.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1B.txt:24]: First one sounds slightly nicer to me when I say it but can't elaborate why

File: script/Ciel/Day 14/14_01_CIEL14_1B.txt [hunk 3]
```diff
@@ -1119,7 +1119,7 @@ I can't allow Arcueid to kill me.
 [sha:01be8f4e9cc388384a8c093bf4e494c635600e3f]{
 -- Page 105, Offset 37338.
 -- 貴方はどんな時でも強がりを言っていた。自分の状態を把握した上で、できもしない事を達成する為に、本気で自分の命を使っていた」
-No matter what happened, you always put on a brave front.
+No matter what happens, you always put on a brave front.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1B.txt:1122]: Speaking about the past, `happened` correct. Put being tricksy by being the same either way.

File: script/Ciel/Day 14/14_01_CIEL14_1C.txt [hunk 4]
```diff
@@ -244,7 +244,7 @@ It'd also free Arcueid and Ciel-senpai from his curse.
 -- Page 157, Offset 37476.
 -- ただし、それは前例のない試みだ。
 // Reading this as "haven't done this experiment before" / can't do a dry run
-However, I'd be gambling on getting it right first time.
+However, I'd be gambling on getting it right the first time.
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1C.txt:247]: "Right first time" is a phrase

File: script/Ciel/Day 14/14_01_CIEL14_1C.txt [hunk 5]
```diff
@@ -341,7 +341,7 @@ It sounded more like she was trying to hint at something incredibly important fo
 [sha:fbbcfa735e301690d6f2140b49ae4f8edbf746b5]{
 -- Page 163, Offset 37493.
 -- 「貴方がその眼をもう一段階深く理解すれば、魂だけを仕留める事だって出来る。
-"If you took your understanding of those eyes to the next level,
+"If you take your understanding of those eyes to the next level,
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1C.txt:344]: I think keep original, it's hypothetical

File: script/Ciel/Day 14/14_01_CIEL14_1C.txt [hunk 6]
```diff
@@ -616,7 +616,7 @@ She struck me square in the chest this time,
 [sha:211dd2395b7f980b5d9e0e90f918c337adf3919d]{
 -- Page 179, Offset 37542.
 -- 結果、俺は背中から地面に落下し、そして、
-As a result, I slam into the ground back-first, and then―――
+As a result, I slam into the ground back first, and then―――
```
Comment (rschlaikjer) [inline script/Ciel/Day 14/14_01_CIEL14_1C.txt:619]: I think this should still have the dash, as in prev pr

File: script/Ciel/Day 14/14_01A_CIEL14A_1.txt [hunk 7]
```diff
@@ -145,7 +145,7 @@ I thought she would always be the woman who had smiled at her irredeemable kille
 [sha:529b92054321c48f31392c47b51f8506a2bc7684]{
 -- Page 6, Offset 37629.
 -- 「だから、<吸|・><血|・><鬼|・>には従わない」
-"And there's no way... I can do what a %{i}vampire%{/i} says!"
+"And there's no way... I would do what a %{i}vampire%{/i} says!"
```
Comment (yk1512) [inline script/Ciel/Day 14/14_01A_CIEL14A_1.txt:148]: I'd

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 14/14_01_CIEL14_1B.txt (hunk 2 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01_CIEL14_1B.txt:24]: Just spider silk is more common but I guess we can make an exception if it sounds nicer
image

File: script/Ciel/Day 14/14_01_CIEL14_1B.txt (hunk 3 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01_CIEL14_1B.txt:1122]: Oh what an obvious error on my part.

File: script/Ciel/Day 14/14_01_CIEL14_1C.txt (hunk 6 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01_CIEL14_1C.txt:619]: Without a dash is more common.
image

Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01_CIEL14_1C.txt:619]: Like all tools, you need to know its limits - if I search a newspaper corpus for 'back first', we get a lot of hits for contexts that _aren't_ the one we're using here:
> The results of the anemia work-up came back first.
> I've got to get the script back first.
> The only bad thing about writing is reading back first drafts.

These are going to skew the results. I think that back-first counts as an adjective here, so it gets compounded with a hyphen

Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01_CIEL14_1C.txt:619]: alright works for me

File: script/Ciel/Day 14/14_01_CIEL14_1B.txt [hunk 8]
```diff
@@ -322,7 +322,7 @@ My throat won't move.
 -- Page 59, Offset 37196.
 -- そんな余分な真似をすれば、次の瞬間、この首が千切られて転がる予感が、全身を縛って離れない。
 I remain frozen in place,
- paralyzed by the intuition that if I do anything so pointless as breathe then my head will be ripped clean off my shoulders.
+ paralyzed by the intuition that if I do anything so pointless as breathe, then my head will be ripped clean off my shoulders.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01_CIEL14_1B.txt:325]: Damn... this sentence really bothers me

File: script/Ciel/Day 14/14_01_CIEL14_1C.txt [hunk 9]
```diff
@@ -82,7 +82,7 @@ Now that I've experienced it firsthand,
 [sha:9ddc325223cd9d313789160132d57257d382e4c5]{
 -- Page 146, Offset 37447.
 -- ここまで自由自在に動ける肉体になってしまったら、人間としての倫理感も価値観も保ってはいられなくなる。
-Anyone would be hard pressed to maintain their human values or ethics when endowed with a body that can move this freely.
+Anyone would be hard-pressed to maintain their human values or ethics when endowed with a body that can move this freely.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01_CIEL14_1C.txt:85]: let's just combine values and ethics into morals

File: script/Ciel/Day 14/14_01_CIEL14_1B.txt (hunk 8 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 14/14_01_CIEL14_1B.txt:325]: any suggestions on what to change it to?

**Pull request merged**